lib/svgo/coa.js

```diff
@@ -300,23 +301,32 @@ function optimizeFolder(config, dir, output) {
  * @param {string} output output directory
  * @return {Promise}
  */
-function processDirectory(config, dir, files, output) {
+async function processDirectory(config, dir, files, output) {
   // take only *.svg files, recursively if necessary
   var svgFilesDescriptions = getFilesDescriptions(config, dir, files, output);
-
-  return svgFilesDescriptions.length
-    ? Promise.all(
-        svgFilesDescriptions.map((fileDescription) =>
-          optimizeFile(
-            config,
-            fileDescription.inputPath,
-            fileDescription.outputPath,
-          ),
-        ),
-      )
-    : Promise.reject(
-        new Error(`No SVG files have been found in '${dir}' directory.`),
-      );
+  
+  if ( svgFilesDescriptions.length == 0 ) {
+    throw 'No SVG files have been found in "' + dir + '" directory.';
+  }
+  
+  let results = [ ],
```

Comment:
Run Prettier to fix some of the failing tests, it will fix the formatting on a couple lines including this one